### PR TITLE
Fix cond

### LINKED ISSUE #55: If else
`class="@@if (active === 'index') {index}"` works fine, but how can I have `else` condition also?

`class="@@if (active === 'index') {index} else {inner}"` throws error, `class="@@if (active === 'index') {index} @@else {inner}"` also.

### LINKED ISSUE #63: Partially string parser

### LINKED ISSUE #64: use @@if and @@include with a param
```  
@@if(this.env==='production') {
    @@include('partials/google-analytics.html.tpl', {"ua": "XXXXXXX"})
}
```
will output the correct partial, with the parameter in place, but it also outputs the `@@if` condition

### PR DESCRIPTION
**Pull request by TrySound**
This is a big victory!
fix #64 
fix #63 
fix #55

### CODE REVIEW

**Comment by coderhaoxin** [comment]
:+1: @TrySound Please wait one or two days. I'm outing. :smile:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-file-include",
-  "version": "0.11.1",
+  "version": "0.11.2",
```
Comment (coderhaoxin) [inline package.json]: It should be v0.12.0.

Comment (coderhaoxin) [inline package.json]: We should make release as separate commit. :smile:

Comment (TrySound) [inline package.json]: @coderhaoxin This PR just fixes bugs.

**Comment by TrySound** [comment]
@coderhaoxin When you will look at this? Want to continue work.

**Comment by coderhaoxin** [comment]
land 33b37b24bc9970e9d9f16ccbcb5c814c8b8a3eff 974f6d1580519e933eb893ff41433b7a25761e5c
